lib/logux.rb

```diff
@@ -66,10 +69,6 @@ def self.process_batch(stream:, batch:)
     Logux::Process::Batch.new(stream: stream, batch: batch).call
   end
 
-  def self.logger
```

Comment:
Why did you decide to remove this method? It looks more straightforward to use just `logger` method. Also, your approach a little bit harder to maintain, in the case when we want to replace the logger with external implementation(in that case, we should replace it on every call, instead of just in one method). I think better to replace the content of this method with a link to your class.

Reply:
Yes, you're right, it's better. I fixed it